src/components/external-apps/ExternalAppService.ts

```diff
@@ -12,6 +12,14 @@ export interface HelmAppDeploymentHistoryResponse extends ResponseType {
     result?: HelmAppDeploymentHistory
 }
 
+export interface HelmAppDeploymentDetails {
```

Comment:
There is another interface "HelmAppDeploymentDetail" in same file. So this name is confusing. 
You can use "HelmAppDeploymentManifestDetail"